src/components/Pages/PlaylistCollection/Page.js

```diff
@@ -125,6 +125,15 @@ const PlaylistCollectionPage = ({ collection, nextLink = null, prevLink = null,
 
   const filterOutUnits = items.map(item => item.unit).filter(u => !!u) || [];
 
+  // Don't recommend lesson preparation, skip to next unit.
+  let recommendUnit = unit;
+  if (recommendUnit.id === Object.entries(collection.ccuNames).find(([unitUid, ccuName]) => ccuName === '0')[0]) {
```

Comment:
Are you sure this long thing will always work? Maybe to put it into try/catch?

Reply:
Good point. Added some more checks.